src/model.ts

```diff
@@ -90,6 +90,13 @@ export class GitExtension implements IGitExtension {
     return this._currentBranch;
   }
 
+  /**
+   * The list of tags in repo
+   */
+  get taglist() {
+    return this._tagList;
+  }
+
```

Comment:
```suggestion
```

You do not set `this._tagList` in any place. And it is better to drop it anyway as it won't be up to date - except if introducing a polling like for the git status. But this is not the goal here. So if the user want the tag list, he will need to call `tags` to get it.